source/about/cloud-subscriptions.rst

```diff
@@ -6,35 +6,19 @@ From 2 February 2023, Mattermost Cloud subscriptions are switching from a monthl
 Buy a Cloud subscription
 ------------------------
 
-Mattermost Cloud can be used free forever with Mattermost Free.
-
-You can also start a free, 30-day trial of Mattermost Enterprise to explore additional features. During or after the trial, you can choose to upgrade to Mattermost Professional in-product or Enterprise by `contacting sales <https://mattermost.com/contact-sales/>`__.
-
-For Professional, follow the steps below:
```

Comment:
@justinegeffen this flow is still valid. Users in Mattermost Cloud can still upgrade to a Professional plan.

Line 25, yes, should be deleted "If you choose not to upgrade at the end of your 30-day trial"